src/main/java/seedu/opus/logic/commands/SortCommand.java

```diff
@@ -22,6 +25,11 @@ public SortCommand(String keyword) {
     @Override
     public CommandResult execute() {
         model.sortList(keyword);
-        return new CommandResult(MESSAGE_SUCCESS);
+        if (keyword.contains("all") || keyword.contains("status") || keyword.contains("priority")
```

Comment:
I believe you could have an array of all the possible keywords and do something like this
http://stackoverflow.com/questions/8992100/test-if-a-string-contains-any-of-the-strings-from-an-array